arbitrum-docs/launch-orbit-chain/how-tos/customize-precompile.mdx

```diff
@@ -406,3 +541,8 @@ We can use the @nitroContractsRepositorySlug@ and @nitroContractsPathToPrecompil
 [precompile_impl_dir_link]: https://github.com/OffchainLabs/@nitroRepositorySlug@/blob/@nitroVersionTag@/@nitroPathToPrecompiles@/
 [precompile_interface_dir_link]: https://github.com/OffchainLabs/@nitroContractsRepositorySlug@/blob/@nitroContractsCommit@/@nitroContractsPathToPrecompilesInterface@/
 [precompile_main_file_link]: https://github.com/OffchainLabs/@nitroRepositorySlug@/blob/@nitroVersionTag@/@nitroPathToPrecompiles@/precompile.go
+[arbosstate_file_link]: https://github.com/OffchainLabs/@nitroRepositorySlug@/blob/@nitroVersionTag@/@nitroPathToArbosState@/arbosstate.go
+[arbosstate_link]: https://github.com/OffchainLabs/nitro/blob/v2.2.0/arbos/arbosState/arbosstate.go#L38
```

Comment:
Can you use @nitroRepositorySlug@ in these 3 lines too?

Reply:
I didn't apply those tags to there 3 lines is because they point to specific line, so I use a constant url here, if we use it, the file will update to the latest version which might be different line in the future

Reply:
Alright, in the future maybe we can generalize a method like the one we use for finding the right line of the precompiles functions => https://github.com/OffchainLabs/arbitrum-docs/blob/master/website/src/scripts/precompile-reference-generator.ts#L69